package.json

```diff
@@ -240,7 +240,7 @@
     "@stylistic/eslint-plugin": "^1.5.0",
     "@swc-node/register": "1.8.0",
     "@swc/cli": "^0.3.12",
-    "@swc/core": "~1.3.100",
+    "@swc/core": "1.7.42",
```

Comment:
SWC core 1.3 was not compatible with lingui ; SWC 1.10 is the latest version but I found it also wasn't compatible. The latest compatible version according to https://plugins.swc.rs/ was the 1.7.42